tests/js/click/click-basic-tests.js

```diff
@@ -0,0 +1,143 @@
+/*
+Copyright (c) 2020 The Gamepad Navigator Authors
+See the AUTHORS.md file at the top-level directory of this distribution and at
+https://github.com/fluid-lab/gamepad-navigator/raw/master/AUTHORS.md.
+
+Licensed under the BSD 3-Clause License. You may not use this file except in
+compliance with this License.
+
+You may obtain a copy of the BSD 3-Clause License at
+https://github.com/fluid-lab/gamepad-navigator/blob/master/LICENSE
+*/
+
+/* global gamepad, jqUnit */
+
+(function (fluid, $) {
+    "use strict";
+
+    $(document).ready(function () {
+
+        fluid.registerNamespace("gamepad");
+        fluid.registerNamespace("gamepad.tests");
+
+        jqUnit.module("Gamepad Navigator Basic Click Tests", {
+            setup: function () {
+                gamepad.tests.windowObject = window;
+                gamepad.tests.count = 2;
+
+                // To test the gamepad's initial state.
+                gamepad.tests.modelAtRest = {
```

Comment:
You should make a common helper function to take care of this and not repeat it in every new fixture.